ldap/cache.go

```diff
@@ -112,6 +113,27 @@ func (c *userCache) Get(token string) (gimlet.User, bool, error) {
 	return u.user, true, nil
 }
 
+func (c *userCache) Clear(u gimlet.User, all bool) error {
+	c.mu.RLock()
```

Comment:
`c.mu.Lock()`, I think, since it's modifying data.

Reply:
omg! :disappointed_relieved: